templates/zerver/footer.html

```diff
@@ -1,186 +1,111 @@
-{% if corporate_enabled %}
-<div class="footer">
-    <div class="footer-section">
-        <h3>{{ _("Product") }}</h3>
-        <ul>
-            <li><a href="/">{{ _("Home") }}</a></li>
-            <li><a href="/why-zulip/">{{ _("Why Zulip") }}</a></li>
-            <li><a href="/features/">{{ _("Features") }}</a></li>
-            <li><a href="/plans/">{{ _("Plans &amp; pricing") }}</a></li>
-            <li><a href="/self-hosting/">{{ _("Self-hosting") }}</a></li>
-            <li><a href="/apps/">{{ _("Desktop &amp; mobile apps") }}</a></li>
-            <li><a href="/security/">{{ _("Security") }}</a></li>
-            <li><a href="/integrations/">{{ _("Integrations") }}</a></li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <h3>{{ _("Solutions") }}</h3>
-        <ul>
-            <li>
+<footer id="footer">
+    {% if corporate_enabled %}
+    <div class='footer__container'>
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("Product") }}
+            </span>
+            <p>
+                <a href="/why-zulip/">{{ _("Why Zulip") }}</a>
+                <a href="/features/">{{ _("Features") }}</a>
+                <a href="/plans/">{{ _("Plans &amp; pricing") }}</a>
```

Comment:
I would expect us to want a normal `&` here and below?

Reply:
Okay, changed.